queries/boundaries.jinja2

```diff
@@ -60,6 +60,7 @@ WHERE
 
 SELECT
   tags->'name' AS name,
+  'openstreetmap.org' AS source,
```

Comment:
Nice catch!

Please also specify Natural Earth as a source at the top of the file in the macro:

```
{% macro ne_boundary_cols() %}
     gid AS __id__,
     'naturalearthdata.com' AS source,
     {% filter geometry %}the_geom{% endfilter %} AS __geometry__,
     mz_calculate_json_boundaries(t.*) AS mz_properties
 {% endmacro %}
```